Extension/src/telemetry.ts

```diff
@@ -64,11 +63,14 @@ export function activate(): void {
             const packageInfo: IPackageInfo = getPackageInfo();
             if (packageInfo) {
                 let targetPopulation: TargetPopulation;
-                const userVersion: PackageVersion = new PackageVersion(packageInfo.version);
-                if (!userVersion.suffix) {
-                    targetPopulation = TargetPopulation.Public;
-                } else if (userVersion.suffix === "insiders") {
+
+                // If insiders.flag is present, consider this an insiders build.
+                // If release.flag is present, consider this a release build.
+                // Otherwise, consider this an internal build.
+                if (util.checkFileExistsSync(util.getExtensionFilePath("insiders.flag"))) {
```

Comment:
Can you promote this into a utility function? Other places in the code may want to know whether we are pre-release or not in the future and it would be nice not to have to remember the filenames in multiple locations.